02 Requiements/UseCases/UseCase1SendRequest(Fully).md

```diff
@@ -0,0 +1,17 @@
+
+UseCase navn | Send Request | 
+-------------| -------------------------------| 
+Scope        | NY-builder
+Primary Actor| Consultant
+Level        | User goal
+Beskrivelse  | The Consultant specifies height and width of the needed wall, then a guiding price is generated and is ready to send.
+Stakeholders and Interests |<br> <ul><li>Consultant (wishes to make the customer an offer)</li><li>NewYorker (potential recipient of an order)</li><li>Customer (The consumer who wishes to purchase a construct for their home)</li></ul>
+Preconditions |<br> <ul><li>An estimate width and height of the wall to be produced. </li><li> Width and height adheres to the productions limitations.</li></ul>
+Succes Guarantee<br>(Postconditions) | A request for a wall is sent to NewYorker for production.
+Main Succece Scenario |  1. Consultant defines wall height<br><br> 2. Consultant defines Wall width <br><br> 3. The application presents an offer for the wall<br><br> 4. Consultant makes the customer fill out contact information <br><br> 5. Consultant sends a production request to NewYorker and a copy to the customer.
```

Comment:
the fifth point is it only the NY and customers that would get this email because I think also the so-called consultant would also get an email.